bin/kuzzle-createFirstAdmin.js

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/env node
+var
+  readlineSync = require('readline-sync'),
+  clc = require('cli-color'),
+  request = require('request-promise'),
+  name,
+  password,
+  step = 0,
+  // steps definitions
+  steps = [
+  	{
+      // get username
+  		prompt: '\n[❓] Which username for the first admin ?\nTip: avoid to name it admin, or root...\n: ',
+  		check: function(userEntry) {
+  			var _name = userEntry.trim(),
+  			badNames = ['admin','root'];
+  			if (badNames.indexOf(_name) === -1) {
```

Comment:
- `admin` is a 5-characters length username
- why are we explicitly allowing `root` as the admin name if we say that it should be avoided?

I would lower the size limit to 3 characters and remove the name checking altogether.